kubernetes/overlays/eservice-descriptors-archiver/dev/configmap.yaml

```
@@ -8,4 +8,7 @@ data:
   CATALOG_PROCESS_URL: "http://{{CATALOG_PROCESS_SERVICE_NAME}}:{{BACKEND_SERVICE_PORT}}/{{CATALOG_PROCESS_APPLICATION_PATH}}/{{CATALOG_PROCESS_INTERFACE_VERSION}}"
   AGREEMENTS_COLLECTION_NAME: "{{AGREEMENTS_COLLECTION_NAME}}"
   ESERVICES_COLLECTION_NAME: "{{ESERVICES_COLLECTION_NAME}}"
-        
\ No newline at end of file
+  JWT_ISSUER: "{{ESERVICE_DESCRIPTORS_ARCHIVERJWT_ISSUER}}"
```

I think it's fine in this case to use
```suggestion
  JWT_ISSUER: "{{INTERNAL_JWT_ISSUER}}"
```
and remove the variable from other scripts